design/ceph/object/store.md

```diff
@@ -73,7 +74,124 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
+
+```yaml
+  radosNamespaces:
+    metadataPoolName: rgw-meta-pool
+    dataPoolName: rgw-data-pool
+```
+
+To create the pools that will be shared by multiple object stores, create the following CephBlockPool CRs:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-meta-pool
+  spec:
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+ metadata:
+    name: rgw-data-pool
+  spec:
+    failureDomain: osd
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 2
+    application: rgw
+```
+
+The `.rgw.root` pool must be defined before the object store is created. This pool is used for storing the root info for all object stores.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-root
+  spec:
+    name: .rgw.root
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+```
+
+The pools for this configuration will be created as below:
+
+```bash
+# ceph osd pool ls
+.rgw.root
+rgw-meta-pool
+rgw-data-pool
+```
+
+And the pool configuration in zone is as below:
+
+```json
+# radosgw-admin zone get --rgw-zone=my-store
+{
+    "id": "2220eb5f-2751-4a51-9c7d-da4ce1b0e4e1",
+    "name": "my-store",
+    "domain_root": "rgw-meta-pool:my-store.root",
+    "control_pool": "rgw-meta-pool:my-store.control",
+    "gc_pool": "rgw-meta-pool:my-store.gc",
+    "lc_pool": "rgw-meta-pool:my-store.lc",
+    "log_pool": "rgw-meta-pool:my-store",
+    "intent_log_pool": "rgw-meta-pool:my-store.intent",
+    "usage_log_pool": "rgw-meta-pool:my-store.usage",
+    "roles_pool": "rgw-meta-pool:my-store.roles",
+    "reshard_pool": "rgw-meta-pool:my-store.reshard",
+    "user_keys_pool": "rgw-meta-pool:my-store.users.keys",
+    "user_email_pool": "rgw-meta-pool:my-store.users.email",
+    "user_swift_pool": "rgw-meta-pool:my-store.users.swift",
+    "user_uid_pool": "rgw-meta-pool:my-store.users.uid",
+    "otp_pool": "rgw-meta-pool:my-store.otp",
+    "system_key": {
+        "access_key": "",
+        "secret_key": ""
+    },
+    "placement_pools": [
+        {
+            "key": "default-placement",
+            "val": {
+                "index_pool": "rgw-data-pool:my-store.buckets.index",
+                "storage_classes": {
+                    "STANDARD": {
+                        "data_pool": "rgw-data-pool:my-store.buckets.data"
+                    }
+                },
+                "data_extra_pool": "rgw-data-pool:my-store.buckets.non-ec",
+                "index_type": 0,
+                "inline_data": "true"
+            }
+        }
+    ],
+    "realm_id": "65a7bf34-42d3-4344-ac40-035e160d7f9e",
+    "notif_pool": "rgw-meta-pool:my-store.notif"
```

Comment:
@yuvalif will changing this affect anything related to your notifications work?

Reply:
the default pool name for notifications is handcoded (similarly t otoher pools), but you can set it with the zone parameters. so, if it is set, there should be no issue

Reply:
Thanks, Yuval. Sounds like it might affect that work then, so we should note that in the design.

Reply:
Does this line already define the `notif` rados namespace, or what else is needed to document?